src/redux/authReducer.js

```diff
@@ -0,0 +1,33 @@
+const initialAuthState = {
+  token: null,
+  user: null,
+};
```

Comment:
Since there is no use of initialAuthState in reducer file, I guess it would be appropriate to have it in context since we are using it over there only.

Reply:
The idea was to have the state and dispatch together in a single file. I'm currently not sure how this is beneficial but I think that it helps in understanding the code well as we have state and dispatch together in a single file. For now, if you permit we could let it be the same and then would confirm it once from some MA and then do the change if needed to do so. Does this work @AtulPant2704 ?